README.md

```diff
@@ -70,7 +70,8 @@ Before checking-in commits, run: `make ci`, which is simlar to the ci.yml workfl
 ## Stay informed on progress
 
 * **Changelog**: See [CHANGELOG.md](CHANGELOG.md) for the latest updates
-* **Chat**: Join the [Focalboard community channel](https://community.mattermost.com/core/channels/focalboard)
+* **Forum**: Join the [Focalboard public forum](https://forum.mattermost.org/t/focalboard-discussion/11377) for open discussion, troubleshooting and comments
```

Comment:
We are thinking about focusing for now in the Github Dicussions. @coreyhulen @chenilim what do you think about this?

Reply:
For developers and potential contributors, I think we want to have people on GitHub to add stars and to work in issues and PRs. 

For end users and IT admins I think we want them on Mattermost forums with general end user feedback, troubleshooting, tips & tricks, templates, etc. We want to control the experience and navigation bar, and guide that audience to the right place. Also, we want SEO from people linking to solutions in our troubleshooting forum on a Mattermost web property.

Reply:
@jespino updated for GitHub Discussions, just looked and it's already active :)